src/App.tsx

```diff
@@ -55,7 +56,7 @@ class App extends React.Component<AppProps> {
   }
 }
 
-const mapStateToProps = (state: any) => {
+const mapStateToProps = (state: any, ownProps: any = {}) => {
```

Comment:
Is `ownProps: any = {}` something that needs to be included in components that use mapStateToProps?

Reply:
i tried to respond to this via email but it just left a comment on the PR so incase you missed it "oops no this was from when i was testing stuff out, will remove", i removed this in a new commit